internal/rego/oci/oci.go

```diff
@@ -73,6 +74,43 @@ func registerOCIBlob() {
 	})
 }
 
+func registerOCIDescriptor() {
+	manifest := types.NewObject(
+		[]*types.StaticProperty{
+			// Specifying the properties like this ensure the compiler catches typos when
+			// evaluating rego functions.
+			{Key: "mediaType", Value: types.S},
```

Comment:
Either we add all properties from the [Descriptor](https://pkg.go.dev/github.com/google/go-containerregistry@v0.20.2/pkg/v1#Descriptor) here or we call this function `ec.oci.descriptor_media_type`

Reply:
I missed that here. I was setting them within the `ociDescriptor` function.